devinfo/src/mountinfo/error.rs

```diff
@@ -65,3 +65,9 @@ impl From<semver::Error> for MountInfoError {
         Self::Semver(semver_err)
     }
 }
+
+impl From<MountInfoError> for std::io::Error {
+    fn from(err: MountInfoError) -> Self {
+        std::io::Error::new(std::io::ErrorKind::Other, err)
```

Comment:
should we match on the various error types and build a more precise error?

Reply:
Oh, I'm sorry that was lazy of me. Sure!

Reply:
Yeah this was the whole reason why I suggest to do this, so we can build a "better" error